qark/plugins/helpers.py

```diff
@@ -88,12 +96,14 @@ def run_regex(filename, rex):
             for curr_line in f:
                 if re.search(rex, curr_line):
                     # exclude everythingin EXCLUDE_REGEXES
-                    if re.match(EXCLUSION_REGEX, curr_line):
-                        pass
-                    else:
+                    if not re.match(EXCLUSION_REGEX, curr_line):
                         things_to_inspect.append(curr_line)
     except IOError:
-        log.debug("Unable to read file: %s results will be inaccurate", filename)
+        log.debug("Unable to open file: %s results will be inaccurate", filename)
+    except UnicodeDecodeError:
+        pass
```

Comment:
Shouldn't we log here too?

Reply:
This is when regex is trying to run across files of weird types, like images. There is going to be a lot of output from this log but it is a debug i'm okay with adding it.